internal/pkg/describe/lb_web_service.go

```diff
@@ -268,6 +367,7 @@ type serviceDiscoveries []*ServiceDiscovery
 
 func (s serviceDiscoveries) humanString(w io.Writer) {
 	fmt.Fprintf(w, "  %s\t%s\n", "Environment", "Namespace")
+	fmt.Fprintf(w, "  %s\t%s\n", "-----------", "---------")
```

Comment:
Can we also do the headers magic for these remaining columns for consistency?

Reply:
yes, duh. done!